js/ui/grid_core/ui.grid_core.keyboard_navigation.js

```diff
@@ -494,7 +494,7 @@ const KeyboardNavigationController = core.ViewController.inherit({
     _tabKeyHandler: function(eventArgs, isEditing) {
         const editingOptions = this.option('editing');
         const direction = eventArgs.shift ? 'previous' : 'next';
-        const isCellPositionDefined = isDefined(this._focusedCellPosition) && !isEmptyObject(this._focusedCellPosition);
+        const isCellPositionDefined = isDefined(this._focusedCellPosition) && !isEmptyObject(this._focusedCellPosition) && !(this._focusedCellPosition.rowIndex === -1 && this._focusedCellPosition.columnIndex === -1);
```

Comment:
```suggestion
        const isCellPositionDefined = 
                isDefined(this._focusedCellPosition) &&
                !isEmptyObject(this._focusedCellPosition) &&
                !(
                        this._focusedCellPosition.rowIndex === -1 &&
                        this._focusedCellPosition.columnIndex === -1
                );
```

Reply:
or maybe it's better to move to helper function, if similar expression is used somewhere else